toptal/picasso
# feat: change number input icon

[FX-1089](https://toptal-core.atlassian.net/browse/FX-1089)

### Description

Changed the icon from a custom one to Picasso's `ArrowUpMinor16` and `ArrowDownMinor16`

### How to test

- Open `NumberInput` in storybook.
- Verify that `ArrowUpMinor16` and `ArrowDownMinor16` from [Picasso's Icon](https://picasso.toptal.net/?path=/story/components-folder--icons) are used.

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
|![image](https://user-images.githubusercontent.com/17337276/93432046-cb961a80-f8e6-11ea-998e-4cb1d3a0850d.png)|![image](https://user-images.githubusercontent.com/17337276/93431911-9e496c80-f8e6-11ea-9d0e-14779834dd16.png)|

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation


</details>
